src/gui/mainwindow.cpp

```diff
@@ -1058,7 +1058,7 @@ void MainWindow::toggleVisibility(const QSystemTrayIcon::ActivationReason reason
                 return;
 
             // Make sure the window is not minimized
-            setWindowState((windowState() & ~Qt::WindowMinimized) | Qt::WindowActive);
+            setWindowState((windowState() & ~Qt::WindowMinimized) & Qt::WindowActive);
```

Comment:
This is entirely wrong, do you have any idea what this does?

Reply:
And yet it works.